userspace/libsinsp/sinsp.h

```diff
@@ -1323,3 +1328,79 @@ VISIBILITY_PRIVATE
 };
 
 /*@}*/
+
+#define SINSP_LOG_(severity, fmt, ...)                                         \
```

Comment:
are these macros used somewhere?

Reply:
They are used somewhere in the libs. I didn't remove them just to be conservative: maybe some libs consumer uses them. But, if we all agree, I can remove them :)

Reply:
uhm got it! I would be on board to remove them, WDYT @falcosecurity/libs-maintainers? In the worst scenario, we could re-introduce them if some adopter wants to rely on them

Reply:
Move them to a separate header? We can include it from sinsp.h or not (or remove it after a while and if nobody complains, remove the file entirely?)